packages/cryptoplease/lib/app/screens/authenticated/profile/components/danger_section.dart

```diff
@@ -13,6 +13,7 @@ class DangerSection extends StatelessWidget {
           sl<ISKPRepository>().clear();
           sl<ODPRepository>().clear();
           sl<OSKPRepository>().clear();
+          sl<IntercomService>().logout();
```

Comment:
This should rather be a deprecated way of having it here. Check `LogoutListener` and its usages instead.